packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -159,17 +159,81 @@ similar to (for example) `inputPath`.
 
 #### Lambda example
 
+[Invoke](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html) a Lambda function.
+
+You can specify the input to your Lambda function through the `payload` attribute.
+By default, no payload is specified so Step Functions invokes Lambda with the empty
+object `{ }` as input.
+
+The following snippet invokes a Lambda Function with the task context as the input
+by referencing the `$` path.
+
+```ts
+new sfn.Task(this, 'Invoke with task context', {
+  task: new tasks.RunLambdaTask(myLambda, {
+    payload: sfn.TaskInput.fromDataAt('$'),
+  }),
+});
+```
+
+When a function is invoked, the Lambda service sends back the following
+[response elements](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseElements)
+
+⚠️ The response from the Lambda function is in an attribute called `Payload`
+
+The following snippet invokes a Lambda Function by referencing the `$.Payload` path
+to reference the output of a Lambda executed before it.
+
+```ts
+new sfn.Task(this, 'Invoke with task context', {
+  task: new tasks.RunLambdaTask(myLambda),
+});
+
+new sfn.Task(this, 'Invoke with output from another Lambda', {
```

Comment:
How about `Invoke with output from the previous task` or `Invoke with the Payload field in the state input`?

It's kind of presented here as a generic way to invoke Lambda but only applies if you've structured your states exactly in this way. I can see users who don't understand input/output processing to just expect it to work all the time. The second option makes sense on its own.